File: Algorithms/Sort/bubble-sort.c
#include <stdio.h>
int main()
{
  int i,j,sz,temp;
  printf("enter the number of elements to input:\n");
  scanf("%d",&sz);
  int ar[sz];
  printf("enter the elments:\n");
  for(i=0;i<sz;i++)
    scanf("%d",&ar[i]);
  printf("array before sorting:\n");
  for(i=0;i<sz;i++)
    printf("%d\n",ar[i]);
  for(i=0;i<sz-1;i++)
  {
    for(j=0;j<sz-1-i;j++)
    {
      if(ar[j]>ar[j+1])
      {
        temp=ar[j];
        ar[j]=ar[j+1];
        ar[j+1]=temp;
      }
    }
  }
  printf("array after sorting:\n");
  for(i=0;i<sz;i++)
    printf("%d\n",ar[i]);
}

File: Algorithms/Sort/bubble-sort/bubble-sort.py
#!/usr/bin/env python3
# Written by @daydreamer at Oct 12, 2016. Trying to do same behaviour as the C code
# Except for the input part
sz = int(input('enter the number of elements to input:\n'))
ar = list(map(int, input('enter the elements:\n').split(' ')))

print("array before sorting:")
for item in ar:
    print(item)

for i in range(sz-1):
    for j in range(sz-1-i):
        if ar[j] > ar[j+1]:
            ar[j], ar[j+1] = ar[j+1], ar[j]

print("array after sorting:")
for item in ar:
    print(item)
